### PR TITLE
Update epytope and Ensembl reference handling

### PR DESCRIPTION
**Pull request by christopher-mohr**
This 
- updates the `epytope` version to the most recent and custom container
- changes the Ensembl (biomart) reference handling, the default reference is now the most recent for `GRCh37` and `GRCh38` respectively. In addition, users can now provide any valid Ensembl reference (archive) version. 
- changes the parameter name and description of the Ensembl reference parameter
- moves the handling of the parameter from the python script to `modules.config` 
- ensures that methods from `epytope` are used if available and remove corresponding code in `epaa.py`

> **Note**
> The handling of the most recent Ensembl (biomart) version is now possible due to the functionality in `epytope` available with the most recent version. This was implemented in a way to be as adaptive as possible to DB schema changes on the Ensembl BioMart side (retrieval of field names via API etc), however with old archive versions (e.g. 2014 and 2018) there are still issues due to inconsistencies on the Ensembl side. 

I will still update to the most recent template to make the linting checks happy. :)

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/epitopeprediction _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -16,12 +16,14 @@ You will need to create a samplesheet with information about the samples you wou
...
-The pipeline currently accepts three different types of input that are genomic variants, peptides and proteins. The supported file formats for genomic variants are `.vcf`, `.vcf.gz` and `tsv`.
-
-:warning: Please note that genomic variants have to be annotated. Currently, we support variants that have been annotated using [SnpEff](http://pcingola.github.io/SnpEff/). Support for [VEP](https://www.ensembl.org/info/docs/tools/vep/index.html) will be available with one of the upcoming versions.
+The pipeline currently accepts three different types of input that are genomic variants, peptides and proteins.
 
 #### Genomic variants
 
+The supported file formats for genomic variants are `.vcf`, `.vcf.gz` and `tsv`.
+
+:warning: Please note that genomic variants have to be annotated. Currently, we support variants that have been annotated using [SnpEff](http://pcingola.github.io/SnpEff/). Support for [VEP](https://www.ensembl.org/info/docs/tools/vep/index.html) will be available with one of the upcoming versions.
```
Comment (jonasscheid) [inline docs/usage.md:25]: usage.md also needs to be updated then

File: nextflow_schema.json [hunk 2]
```diff
@@ -48,12 +48,11 @@
...
             "properties": {
-                "genome_version": {
+                "genome_reference": {
                     "type": "string",
-                    "default": "GRCh37",
-                    "help_text": "This defines against which human reference genome the pipeline performs the analysis including the incorporation of genetic variants e.g..",
-                    "enum": ["GRCh37", "GRCh38"],
-                    "description": "Specifies  the human reference genome version."
+                    "default": "grch37",
+                    "help_text": "This defines against which human Ensembl genome reference the pipeline performs the analysis including the incorporation of genetic variants e.g.. If `grch37` or `grch37` are specified the most recent Ensembl Biomart version for genome versions will be used. Alternatively, an Ensembl Biomart (archive) version can be specified, e.g. http://jan2020.archive.ensembl.org/.",
```
Comment (jonasscheid) [inline nextflow_schema.json:54]:
```suggestion
                    "help_text": "This defines against which human Ensembl genome reference the pipeline performs the analysis including the incorporation of genetic variants e.g.. If `grch37` or `grch38` are specified the most recent Ensembl Biomart version for genome versions will be used. Alternatively, an Ensembl Biomart (archive) version can be specified, e.g. http://jan2020.archive.ensembl.org/.",
```

File: nextflow_schema.json [hunk 3]
```diff
@@ -68,6 +67,22 @@
             "fa_icon": "fas fa-dna",
             "description": "Reference genome related files and options required for the workflow.",
             "properties": {
+                "genome": {
```
Comment (jonasscheid) [inline nextflow_schema.json:70]: Do wen need this parameter?

**Review by jonasscheid — CHANGES_REQUESTED** [review]
Thanks for incorporating this 🚀 
I only see some minor rendering issues, else lgtm

File: nextflow_schema.json (hunk 3 above)
Comment (christopher-mohr) [reply nextflow_schema.json:70]: We don't use it but it would get reintroduced by the template anyway I think and we can ignore it.

**Review by jonasscheid — APPROVED** [review]

**Pull request merged**